tst/testinstall/coll.tst

```diff
@@ -97,15 +97,16 @@ Error, cannot test whether <C> contains the family of its elements
 # Size
 #
 
-# immediate method for collections knowing they are infinite
-gap> c2:=ConjugacyClass(F, F.1);;
-gap> HasSize(c2);
-false
-gap> SetIsFinite(c2, false);
-gap> HasSize(c2);
-true
-gap> Size(c2);
```

Comment:
This test foremost was meant to test that `Size` method. Since it is still there, just not immediate, we could keep the test, and just remove the two checks for `HasSize`.

Reply:
OK.